projects/core/src/occ/utils/interceptor-util.ts

```diff
@@ -2,6 +2,7 @@ import { HttpHeaders, HttpRequest } from '@angular/common/http';
 
 export const USE_CLIENT_TOKEN = 'cx-use-client-token';
 export const USE_CUSTOMER_SUPPORT_AGENT_TOKEN = 'cx-use-csagent-token';
+export const TOKEN_REVOCATION = 'cx-token-revocation';
```

Comment:
we haven't done this before, but I'd be keen to rename this to `TOKEN_REVOCATION_HEADER`

Reply:
I agree. It is renamed to `TOKEN_REVOCATION_HEADER`

Reply:
I agree. It is renamed to `TOKEN_REVOCATION_HEADER`